orcoursetrion/tests/base.py

```diff
@@ -0,0 +1,41 @@
+# -*- coding: utf-8 -*-
+"""
+Test base class with commonly used methods and variables
+"""
+import json
+import unittest
+
+
+class TestGithubBase(unittest.TestCase):
+    """Test Github actions and backing library."""
+
+    OAUTH2_TOKEN = '12345'
+    ORG = 'NOT_REAL'
+    URL = 'http://localhost/'
+
+    TEST_COURSE = 'devops.001'
+    TEST_TERM = 'Spring_2999'
+    TEST_DESCRIPTION = 'foo'
+    TEST_PREFIX = 'testo'
+    TEST_REPO = '{0}-{1}-{2}'.format(
+        TEST_PREFIX, TEST_COURSE.replace('.', ''), TEST_TERM
+    )
+
+    def callback_repo_check(self, request, uri, headers, status_code=404):
+        self.assertEqual(
+            request.headers['Authorization'],
+            'token {0}'.format(self.OAUTH2_TOKEN)
+        )
```

Comment:
This assert in this method (callback_repo_check) is used for verifying that the tested code fires correct requests to github, right?